clusterloader2/run-e2e.sh

```diff
@@ -34,4 +34,4 @@ if [[ "${DEPLOY_GCI_DRIVER:-false}" == "true" ]]; then
 fi
 
 cd ${CLUSTERLOADER_ROOT}/ && go build -o clusterloader './cmd/'
-./clusterloader --alsologtostderr --v=2 "$@"
+./clusterloader --alsologtostderr --v=2 --node-preload-images=gcr.io/k8s-testimages/probes:v0.0.4 "$@"
```

Comment:
Can we make the version of the image from somewhere else (so that it won't get outdated with the nearest bump of it)?

Reply:
Wojtek, this is a test PR just to trigger and debug the image-preload. I'm not going to submit this.